sys/posix/netdb/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = posix_netdb
```

Comment:
Please only squash, when the review is done, so the reviewer can follow the changes. ;)